### ISSUE TITLE
Discussion: various ideas

### ISSUE THREAD

**Issue by jaromeyer** [isaakhanimann/psychonautwiki-journal-android#39]
- replace search icon in the navbar with a pill icon or similar
- substance screen: replace expandable cards with normal cards since there is not that much content anyway and vertical scrolling should IMO be the main way of interacting with mobile UIs
-  is the weird fullscreen graph when tapping effect timeline intentional?
- make descriptions for substance categories popup dialog instead of separate page

**Comment by isaakhanimann** [comment]
I agree with everything. The separate timeline screen is for zooming if the timeline becomes crowded.
